libs/vkd3d/state.c

```diff
@@ -3353,44 +3353,59 @@ static HRESULT vkd3d_bindless_state_add_binding(struct vkd3d_bindless_state *bin
     VkDescriptorSetLayoutBindingFlagsCreateInfoEXT vk_binding_flags_info;
     const struct vkd3d_vk_device_procs *vk_procs = &device->vk_procs;
     VkDescriptorSetLayoutCreateInfo vk_set_layout_info;
-    VkDescriptorSetLayoutBinding vk_binding_info;
-    VkDescriptorBindingFlagsEXT vk_binding_flags;
+    VkDescriptorSetLayoutBinding vk_binding_info[9];
```

Comment:
Why 9?

Reply:
`VKD3D_BINDLESS_SET_MAX_EXTRA_BINDINGS + 1`, the last one being the descriptor array.